.github/workflows/test.yml

```diff
@@ -169,15 +169,25 @@ jobs:
       - run: ruby -v
 
   testLatestRubygemsVersion:
-    name: "Test rubygems: latest upgrades the default RubyGems version"
+    name: "Ruby ${{ matrix.ruby }}: latest upgrades RubyGems"
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - { ruby: '3.2', rg_vers: '3.5.3'  }
```

Comment:
rg_vers -> rubygems_version/expected_version/expected_rubygems_version (abbreviations are an overhead on readability if not super well known ones)